packages/react/src/Dialog/Dialog.tsx

```diff
@@ -626,12 +626,16 @@ const StyledFooter = toggleStyledComponent(
     box-shadow: 0 -1px 0 ${get('colors.border.default')};
     padding: ${get('space.3')};
     display: flex;
-    flex-flow: wrap;
-    justify-content: flex-end;
+    flex-direction: row-reverse;
+    flex-wrap: wrap-reverse;
     gap: ${get('space.2')};
     z-index: 1;
     flex-shrink: 0;
 
+    @media (max-height: 256px) {
+      flex-wrap: nowrap;
+      overflow-x: scroll;
+    }
```

Comment:
```suggestion
    flex-flow: wrap;
    justify-content: flex-end;
    gap: ${get('space.2')};
    z-index: 1;
    flex-shrink: 0;

    @media (max-height: 356px) {
      flex-wrap: nowrap;
      overflow-x: scroll;

      flex-direction: row;
      justify-content: unset;
    }
```

Curious what you think about this change? I think this also means we don't have to utilize `reverse()` on the footer buttons.

Reply:
I like it, it doesn't wrap the same way but I think it's valid and it is simpler!